app/assets/javascripts/discourse/templates/modal/login.hbs

```diff
@@ -32,6 +32,21 @@
             </tr>
           </table>
         </div>
+        <div id="second-factor" style="display: none;">
```

Comment:
This part of the template seems to be duplicated. You should use an ember component to avoid repeating yourself.